src/components/TreeWalker/tests/index.test.tsx

```diff
@@ -1,6 +1,7 @@
 import { mount } from 'enzyme';
 import flushPromises from 'flush-promises';
 import React from 'react';
+import { act } from 'react-dom/test-utils';
```

Comment:
Decided to fix some test errors by using act() whenever changes to the DOM in tests happen.